docs/releases/v2018.1.rst

```diff
@@ -0,0 +1,285 @@
+  to distinguish mesh traffic from unrelated packets.
+
+As enabling VXLAN encapsulation will prevent wired mesh communication with old nodes
+that do not support VXLAN yet, VXLANs can be enabled per-domain using the site configuration
+setting *mesh.vxlan*. VXLAN is enabled by default in multidomain setups; in single-domain
+site configurations, the *mesh.vxlan* setting is mandatory. We recommend to enable
+VXLAN encapsulation in all new sites and domains.
+
+Router advertisement filtering
+==============================
+
+Similar to the builtin batman-adv gateway feature for IPv4, the *gluon-radv-filterd* package
+(*radv-filterd* feature flag) allows to filter IPv6 router advertisements from the mesh
+so that only the RAs with the best routing metric (TQ) reach the clients, ensuring that
+the "best" gateway is chosen as the IPv6 default route.
+
+At the moment, this feature only filters RAs forwarded to clients; the RAs handled on
+the nodes themselves will be unfiltered, so the nodes will still use arbitrary default
+gateways.
+
+IGMP/MLD segmentation
+=====================
+
+The IGMP/MLD segmentation feature previously provided by the *gluon-ebtables-segment-mld*
+package has been extended and moved into the Gluon core; it does not exist as a separate package
+anymore.
+
+Filtering IGMP/MLD queries directed towards the mesh ensures that each node becomes the multicast querier
+for its own clients (unless there are other multicast-aware switches connected to the node), rather
+than electing a single, basically arbitrary node in the mesh to become the querier. Overall,
+this should significantly improve the reliablity of multicast in the mesh. This is especially
+important for IPv6, as the IPv6 Neighbour Discovery Protocol (NDP) is based on local multicast.
+
+See also the documentation of the :ref:`site.conf mesh section <user-site-mesh>`.
+
+Public key in respondd data (optional)
+======================================
+
+If desired, the fastd public key of a node can be included in the respondd nodeinfo data,
+faciliating the correlations of VPN peers and nodes. As the VPN key is transmitted unencrypted
+in the fastd handshake, this would theoretically allow an ISP to determine which nodes
+are operated behind which internet line. Therefore, this feature must be enabled explicitly
+by setting *mesh_vpn.pubkey_privacy* to ``false`` in *site.conf*.
+
+B.A.T.M.A.N. V (experimental)
+=============================
+
+When using batman-adv compat 15, it is now possible to switch to the new routing
+algorithm B.A.T.M.A.N. V (while the old algorithm is called B.A.T.M.A.N. IV) by
+setting *mesh.batman_adv.routing_algo* to ``"BATMAN_V"``. Note that the new routing
+algorithm is not backwards-compatible, so nodes using different algorithms can
+not interoperate.
+
+Site changes
+************
+
+site.mk
+=======
+
+TODO WIP
+
+* Feature flags
+
+site.conf
+=========
+
+TODO WIP
```

Comment:
maybe we could also add the removal of the contact field mandatory option, although it was removed in v2017.1.8, too. currently, upgrading to v2017.1.8 is not considered mandatory before upgrading to v2018.1 therefore we should add this here.

Reply:
How about: "The option to obligatorily require entering contact-data in config mode was removed. Providing contact data is entirely optional now."